src/navigation.md

```diff
@@ -189,6 +189,7 @@ search:
         - [Flows with demands](graph/flow_with_demands.md)
         - [Minimum-cost flow](graph/min_cost_flow.md)
         - [Assignment problem](graph/Assignment-problem-min-flow.md)
+        - [All-pairs minimum cut - Gomory Hu](graph/gomory_hu.md)
```

Comment:
```suggestion
        - [Gomory-Hu tree](graph/gomory_hu.md)
```

Reply:
Please also add it to the list of new articles in [README](https://github.com/cp-algorithms/cp-algorithms/blob/master/README.md).